test/sharness/Rules.mk

```diff
@@ -52,7 +61,7 @@ $(d)/deps: $(SHARNESS_$(d)) $$(DEPS_$(d)) # use second expansion so coverage can
 test_sharness_deps: $(d)/deps
 .PHONY: test_sharness_deps
 
-test_sharness_short: $(d)/aggregate
+test_sharness_short: $(d)/aggregate $(d)/junit_xml
```

Comment:
Most users won't need the JUnit xmls so there is no reason to create them for them.